services/dy-static-file-server/docker/custom/Dockerfile

```diff
@@ -3,18 +3,35 @@ FROM nginx:1.21.0-alpine as base
 #
 #  USAGE:
 #     cd services/dy-static-file-server
-#     docker build -f Dockerfile -t dy-static-file-server:prod --target production ../../
+#     docker build -f docker/custom/Dockerfile -t dy-static-file-server:prod  .
 #     docker run dy-static-file-server:prod
 #
 
 LABEL maintainer=sanderegg
 COPY docker/custom/nginx.conf /etc/nginx/nginx.conf
 COPY docker/custom/boot.sh /boot.sh
 
-COPY src/dy_static_file_server/hello-world.txt /www/hello-world.txt
+RUN apk add --update --no-cache "python3=3.8.10-r0" py3-pip
```

Comment:
r0 ? what is this? is that the official release name?
Also here it would be better to have something like
```Dockerfile
ARG PYTHON_VERSION="3.8.10"
RUN apk add --update --no-cache \
         ${PYTHON_VERSION} \
         py3-pip
```